modules/core/shared/src/main/scala/retry/RetryPolicies.scala

```diff
@@ -3,7 +3,7 @@ package retry
 import java.util.concurrent.TimeUnit
 
 import cats.Applicative
-import cats.syntax.functor._
+import cats.implicits._
```

Comment:
Do you mind if we avoid using `implicits._`? Usually I prefer importing the things I need explicitly, unless it results in a ridiculous number of imports.